src/components/deviceCPULoad.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState, useEffect } from 'react'
+
+import StandartLineChart from './StandardLineChart'
+import ShadowBox from './ShadowBox'
+import IconText from './IconText'
+import { Paragraph, Separator, XStack, YStack } from 'tamagui'
+
+const data = [
+  {
+    id: 'cpu',
+    color: '#8DC6BC',
+    data: [
+      {
+        x: '1',
+        y: 15,
+      },
+      {
+        x: '2',
+        y: 12,
+      },
+      {
+        x: '3',
+        y: 43,
+      },
+      {
+        x: '4',
+        y: 20,
+      },
+      {
+        x: '5',
+        y: 60,
+      },
+      {
+        x: '6',
+        y: 5,
+      },
+      {
+        x: '7',
+        y: 15,
+      },
+      {
+        x: '8',
+        y: 12,
+      },
+      {
+        x: '9',
+        y: 43,
+      },
+      {
+        x: '10',
+        y: 20,
+      },
+      {
+        x: '11',
+        y: 60,
+      },
+      {
+        x: '12',
+        y: 132,
+      },
+    ],
+  },
+]
+const DeviceCPULoad = () => {
+  const [message, setMessage] = useState('')
```

Comment:
Message can be calculated just as a `const message = ...`, you don't need `useState` or `useEffect` here. Calculate the value with exactly the same calculation as you are using in your `useEffect`.

In general, `useEffect` is primarily for subscribing/taking and unsubscribing/releasing resources based on changing input data, which most of our components will not be doing since they take all of their inputs cleanly as props and render exactly the same thing every time based on those inputs.